stdlib/experimental/experimental.flux

```diff
@@ -21,3 +21,20 @@ builtin to
 
 // An experimental version of join.
 builtin join
+
+// Aligns all tables/series to a common start time
```

Comment:
What value is aligned? By reading the code I can see its the `_time` column but that is not clear until I read the code.